.gitignore

```diff
@@ -5,6 +5,8 @@ test/testbootstrap.php
 build/
 vendor/
 *.pem
```

Comment:
Please remove this change. The place for these IDE-specific ignores is in your global `.gitignore`, not in every package, and it has nothing to do with the subject of your PR.